scripts/get_description.py

```diff
@@ -22,7 +22,7 @@
 def handle_error(message, strict=True, verbose_mode=False):
     if strict:
         print(f"[ERROR] {message}")
-        exit(-1)
+        exit(1)
```

Comment:
I'll merge but next time please add some text in the commit message explaining what you changed and why. I see a lot of strange changes in this test and don't know which one of them was the one that solved the bug, and what was the reason for the other changes. I suspect you either solved here 4 different issues, or you just added a lot of unrelated cleanups to this PR.